l10n_it_fatturapa_sale/__manifest__.py

```diff
@@ -0,0 +1,24 @@
+#  Copyright 2020 Simone Rubino - Agile Business Group
+#  License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+{
+    "name": "ITA - Fatturazione elettronica - Integrazione vendite",
+    "summary": "Questo modulo aggiunge alcuni dati per la "
+               "fatturazione elettronica nell'ordine di vendita",
```

Comment:
Nella schermata delle applicazioni lo spazio di visualizzazione è limitato, compatterei la descrizione: 
```suggestion
    "summary": "Aggiunge alcuni dati per la "
               "fatturazione elettronica nell'ordine di vendita",
```

Reply:
Grazie della review! Fatto